lib/common_struct.h

```diff
@@ -137,7 +137,7 @@ struct LogEntry {
   LogEntry* next;  // The next LogEntry pointer
 
   std::string result;  // The execution result of the LogEntry
-  char status;         //
+  char status;         // TODO(Katie): does this need to be thread safe?
```

Comment:
@Steamgjk  Does this need to be an atomic type? It seems like `recordTd` could be reading this value while `processTd` is writing it.